msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -65,7 +65,9 @@ The following image shows how SSO works when a Teams app user attempts to access
 For more information, see [Add code to enable SSO in a tab app](tab-sso-code.md) and [Add code to enable SSO in your bot app](../../../bots/how-to/authentication/bot-sso-code.md).
 
 > [!IMPORTANT]
-> The `getAuthToken()` is valid only for consenting to a limited set of user-level APIs, such as email, profile, offline_access, and OpenId. It isn't used for other Graph scopes such as `User.Read` or `Mail.Read`. For suggested workarounds, see [Extend your app with Microsoft Graph permissions](tab-sso-graph-api.md).
+>
```

Comment:
Did you mean to add a new line here?

Reply:
As per markdown lint a default blank line is added when we have a note with more than one bullet.